issue_comment.go

```diff
@@ -188,6 +193,15 @@ func setHeaderString(logger zerolog.Logger, overallJUnitSuites *reporters.JUnitT
 	return &failedTCReport
 }
 
+// initPodAndCRsLink initialises the FailedTestCasesReport struct's
+// 'podsLink' and 'customResourcesLink' field with the link to the
+// directory where pod logs and generated custom resources are
+// stored, respectively.
+func (failedTCReport *FailedTestCasesReport) initPodAndCRsLink(scanner *prow.ArtifactScanner) {
+	failedTCReport.podsLink = reportingLinkPrefix + bucketName + "/" + scanner.ArtifactDirectoryPrefix + "gather-extra/artifacts/pods/"
+	failedTCReport.customResourcesLink = reportingLinkPrefix + bucketName + "/" + scanner.ArtifactDirectoryPrefix + "redhat-appstudio-gather/artifacts/"
```

Comment:
great idea, just wondering what do you think about [getting the links directly from e2e xml report](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/redhat-appstudio_e2e-tests/1178/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1792679527474794496/artifacts/redhat-appstudio-e2e/redhat-appstudio-report/artifacts/junit.xml)?

Reply:
good idea, the code now fetches the links from junit.xml as you suggested. PTAL